### ISSUE TITLE
Quick fix for Python3

### ISSUE THREAD

**Issue by mcuee** [danielkucera/cyusb3014-breakout#1]
I just made a simple fix here to get the python code working under Python 3.
https://github.com/mcuee/python_usb/blob/master/cyusb_fx3_loopback-test_python3.py

**Comment by mcuee** [comment]
BTW, I also use your codes to create a new code to test the BulkSrcSink FW.
https://github.com/mcuee/python_usb/blob/master/cyusb_fx3_sourcesink_python3.py

**Comment by danielkucera** [comment]
yes, the obvious `print()` ... what is the difference between `thread` and `_thread` ?

**Comment by mcuee** [comment]
Not a python expert but I found the following.
https://stackoverflow.com/questions/36809788/importerror-no-module-named-thread
In Python 3 instead of:
```
import thread
```
Do:
```
import _thread
```

**Comment by mcuee** [comment]
Maybe the better fix is to use threading module.
https://www.tutorialspoint.com/python3/python_multithreading.htm
